src/pudl/transform/vcerare.py

```diff
@@ -389,6 +482,23 @@ def out_vcerare__hourly_available_capacity_factor(
     def _get_year(df, year):
         return df.loc[df["report_year"] == year]
 
+    # Get census vintage to conform the data to.
+    assert int(_core_censuspep__yearly_geocodes.report_year.max()) >= int(
+        raw_vcerare__fixed_solar_pv_lat_upv["report_year"].max()
+    )  # Check these are in sync
```

Comment:
if this fails, does that mean we need to update our copy of census pep? or would something else be wrong?